.bazelci/presubmit.yml

```diff
@@ -186,6 +186,17 @@ tasks:
     test_targets: *default_windows_targets
     soft_fail: yes
     bazel: "rolling"
+  ubuntu2004_bzlmod_only:
+    name: With bzlmod
+    platform: ubuntu2004
+    # See https://bazel.build/external/migration#migration-process
+    # When WORKSPACE.bzlmod is provided, the workspace file is ignored.
+    shell_commands:
+      - "touch WORKSPACE.bzlmod"
+    build_flags:
+      - "--enable_bzlmod"
+    build_targets:
+      - "//tools/runfiles"
```

Comment:
How much could we expand this now / what's the plan to expand this in the future?

Reply:
I already have a PR ready, just wanted to add presubmit tests before sending it out. See #2314.